### PR TITLE
Bundleout Text file update

### LINKED ISSUE #3456: Bundleout.txt Formatting for IMAGE MEASURES SUMMARY
**Description**  

Related to #3227.

The bundleout.txt file generated by jigsaw formats well when the image paths are short. For example:

```
IMAGE MEASURES SUMMARY                                                          
==========================                                                      
                                                                                
                                      Measures                        RMS(pixels)
                              ************************  **************************************
                              |Accepted    |    Total|   |Samples    |    Lines    |    Total|
              I02774003RDR.cub  120              120      0.1323         0.2334         0.1897     
              I28234014RDR.cub  100              100      0.1772         0.2443         0.2134     
              I36344009RDR.cub   84               84      0.1236         0.1710         0.1492     
              I36943008RDR.cub   44               44      0.2194         0.1939         0.2070     
              I45517014RDR.cub  136              136      0.2076         0.2820         0.2476     
              I52634011RDR.cub   41               41      0.2360         0.3065         0.2736     
              I53121006RDR.cub   40               40      0.1176         0.2060         0.1677     
              I53146002RDR.cub   80               80      0.1171         0.1765         0.1498     
              I54582015RDR.cub   51               51      0.1285         0.1498         0.1396     
              I55069016RDR.cub   63               63      0.1368         0.1731         0.1560     
              I55506042RDR.cub   33               33      0.2349         0.3135         0.2770     
                                                                                
Total RMS:                                                0.1730         0.2294         0.2031 
```

When the image path is long, column alignment is not maintained. For example:

```
IMAGE MEASURES SUMMARY                                                          
==========================                                                      
                                                                                
                                      Measures                        RMS(pixels)
                              ************************  **************************************
                              |Accepted    |    Total|   |Samples    |    Lines    |    Total|
/work/users/lweller/Isis3Tests/Jigsaw/BundleOutUpdates/I02774003RDR.cub  120              120      0.1323         0.2334         0.1897     
/work/users/lweller/Isis3Tests/Jigsaw/BundleOutUpdates/I28234014RDR.cub  100              100      0.1772         0.2443         0.2134     
/work/users/lweller/Isis3Tests/Jigsaw/BundleOutUpdates/I36344009RDR.cub   84               84      0.1236         0.1710         0.1492     
/work/users/lweller/Isis3Tests/Jigsaw/BundleOutUpdates/I36943008RDR.cub   44               44      0.2194         0.1939         0.2070     
/work/users/lweller/Isis3Tests/Jigsaw/BundleOutUpdates/I45517014RDR.cub  136              136      0.2076         0.2820         0.2476     
/work/users/lweller/Isis3Tests/Jigsaw/BundleOutUpdates/I52634011RDR.cub   41               41      0.2360         0.3065         0.2736     
/work/users/lweller/Isis3Tests/Jigsaw/BundleOutUpdates/I53121006RDR.cub   40               40      0.1176         0.2060         0.1677     
/work/users/lweller/Isis3Tests/Jigsaw/BundleOutUpdates/I53146002RDR.cub   80               80      0.1171         0.1765         0.1498     
/work/users/lweller/Isis3Tests/Jigsaw/BundleOutUpdates/I54582015RDR.cub   51               51      0.1285         0.1498         0.1396     
/work/users/lweller/Isis3Tests/Jigsaw/BundleOutUpdates/I55069016RDR.cub   63               63      0.1368         0.1731         0.1560     
/work/users/lweller/Isis3Tests/Jigsaw/BundleOutUpdates/I55506042RDR.cub   33               33      0.2349         0.3135         0.2770     
                                                                                
Total RMS:                                                0.1730         0.2294         0.2031     
```
**Example**  
An example output would be the good column alignment seen in the first example and the long paths seen in the second.

**User Story**
- As a user, I can visually parse the bundleout.txt IMAGE MEASURES SUMMARY section so that I can manually assess the quality of image to image registration relationships.

**Acceptance Criteria**
- [ ] Column alignment in the bundleout.txt is maintained regardless of the length of the image path in the column 0 position.

**Product Owner**
- @jlaura

### PR DESCRIPTION
**Pull request by acpaquette**
Updates the bundleout text file to improve readability of the IMAGE MEASURE SUMMERY

## Description
Adds appropriate padding around records in the IMAGE MEASURE SUMMARY table to make the output more human readable.

## Related Issue
Closes #3456 

## Motivation and Context
This makes the bundleout text file more human readable and allows users to get a better grasp of the bundle results.

## How Has This Been Tested?
Tested through the current ISIS test sweet.

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read and agree to abide by the [Code of Conduct](https://usgs-astrogeology.github.io/code/)
- [x] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [x] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

File: isis/src/control/objs/BundleSolutionInfo/BundleSolutionInfo.cpp [hunk 1]
```diff
@@ -1042,16 +1042,34 @@ namespace Isis {
       }
     }
 
+    // Loop over the observations to find the longest file path/name in the
+    // bunch
+    int filePadding = 0;
+
+    for (int i = 0; i < numObservations; i++) {
+      BundleImageQsp bundleImage = m_statisticsResults->observations().at(i)->at(0);
```
Comment (jessemapel) [inline isis/src/control/objs/BundleSolutionInfo/BundleSolutionInfo.cpp]: You should also loop over the images in each observation. In most cases there is only 1, but observation mode can create multiple images

Comment (jessemapel) [reply isis/src/control/objs/BundleSolutionInfo/BundleSolutionInfo.cpp]: This also matches the nested for loops when they are printed out.

Comment (acpaquette) [reply isis/src/control/objs/BundleSolutionInfo/BundleSolutionInfo.cpp]: :+1:

**Review by jessemapel — APPROVED** [review]
Looks good to go. Go ahead and merge once @lwellerastro has signed off.

**Comment by lwellerastro** [comment]
sure...other than what I got from @acpaquette via email, is there something I need to do here @jessemapel other than a thumbs up?

**Comment by jessemapel** [comment]
@lwellerastro Under the "Files changed" tab submit an approving review. It's just a more formal way to indicate that you signed off on this.

**Review by lwellerastro — APPROVED** [review]

**Pull request merged**